Lib/email/utils.py

```diff
@@ -48,6 +48,45 @@
 specialsre = re.compile(r'[][\\()<>@,:;".]')
 escapesre = re.compile(r'[\\"]')
 
+_EMAIL_CONFIG_FILE = "/etc/python/email.cfg"
+
+
+def _use_strict_email_parsing():
+    """Returns True if strict email parsing is not disabled by
+    config file or env variable.
+    """
+    global _cached_strict_addr_parsing
+
+    try:
+        return _cached_strict_addr_parsing
+    except NameError:
+        disabled = bool(os.environ.get("PYTHON_EMAIL_DISABLE_STRICT_ADDR_PARSING"))
+        if disabled:
+            _cached_strict_addr_parsing = False
+            return _cached_strict_addr_parsing
+
+        try:
+            file = open(_EMAIL_CONFIG_FILE)
+        except FileNotFoundError:
+            pass
+        else:
+            with file:
+                import configparser
+                config = configparser.ConfigParser(
```

Comment:
I wanted to suggest SafeConfigParser, but `interpolation=None` does the same, so it's fine.